internal/server/db/cluster/schema.go

```diff
@@ -453,6 +453,15 @@ CREATE TABLE "storage_buckets" (
 	FOREIGN KEY (node_id) REFERENCES "nodes" (id) ON DELETE CASCADE,
 	FOREIGN KEY (project_id) REFERENCES "projects" (id) ON DELETE CASCADE
 );
+CREATE TABLE "storage_buckets_backups" (
```

Comment:
Can you split off the `internal/server/db` changes into a separate `incusd/db: Add storage_buckets_backups table` commit?